model/post.go

```diff
@@ -31,6 +31,7 @@ type Post struct {
 	Id            string          `json:"id"`
 	CreateAt      int64           `json:"create_at"`
 	UpdateAt      int64           `json:"update_at"`
+	EditAt        int64           `json:"edit_at"`
```

Comment:
Oh it looks like emoji reactions update the `UpdateAt` of a post. I think instead of adding `EditAt` you can check the `OriginalId` field. If that field is blank then the post won't have been edited I'm pretty sure